### PR TITLE
Few more automated UI scenarios

### PR DESCRIPTION
**Pull request by bielikb**
### 🎯 Goal

Automate Thierry's scenarios

### ☑️ Contributor Checklist
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] This change follows zero ⚠️ policy (required)
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### CODE REVIEW

File: StreamChatUITestsAppUITests/Tests/ChannelList_Tests.swift [hunk 1]
```diff
@@ -6,6 +6,53 @@ import XCTest
 
 final class ChannelList_Tests: StreamTestCase {
 
+    let message = "message"
+
+    func test_newMessageShownInChannelPreview_whenComingBackFromOffline() {
```
Comment (evsaev) [inline StreamChatUITestsAppUITests/Tests/ChannelList_Tests.swift:11]: Hm, but there's no connectivity stuff happening the scenario, right?

Comment (bielikb) [inline StreamChatUITestsAppUITests/Tests/ChannelList_Tests.swift:11]:
```suggestion
    func test_newMessageShownInChannelPreview_whenComingBackFromChannel() {
```

Comment (bielikb) [reply StreamChatUITestsAppUITests/Tests/ChannelList_Tests.swift]: @evsaev good catch. I just updated the function name ;)

**Review by evsaev — APPROVED** [review]
🚢

**Pull request merged**